content/influxdb/v2.3/influxdb-templates/use.md

```diff
@@ -13,213 +13,778 @@ related:
+| +   | sample-data-bucket | 0000000000000000 | sample_data   | 168h0m0s         |             | implicit    | 0                |
++-----+--------------------+------------------+---------------+------------------+-------------+-------------+------------------+
+|                                                                                                   TOTAL    |        1         |
++-----+--------------------+------------------+---------------+------------------+-------------+-------------+------------------+
+
+TASKS    +add | -remove | unchanged
++-----+-------------------+------------------+-------------------+-------------+-------+
+| +/- |   METADATA NAME   |        ID        |   RESOURCE NAME   | DESCRIPTION | CYCLE |
++-----+-------------------+------------------+-------------------+-------------+-------+
+| +   | fetch-sample-data | 0000000000000000 | Fetch Sample Data |             |       |
++-----+-------------------+------------------+-------------------+-------------+-------+
+|                                                                     TOTAL    |   1   |
++-----+-------------------+------------------+-------------------+-------------+-------+
+
+BUCKETS
++--------------------+------------------+---------------+-----------+--------------------------------+-------------+
+|    PACKAGE NAME    |        ID        | RESOURCE NAME | RETENTION |          DESCRIPTION           | SCHEMA TYPE |
++--------------------+------------------+---------------+-----------+--------------------------------+-------------+
+| sample-data-bucket | d981dd0a2fed16be |  sample_data  | 168h0m0s  |                                |  implicit   |
++--------------------+------------------+---------------+-----------+--------------------------------+-------------+
+|                                                                                 TOTAL              |      1      |
++--------------------+------------------+---------------+-----------+--------------------------------+-------------+
+
+Stack ID: 09bd87cd33be3000
+```
+
+- [Apply a template from the file system](#apply-a-template-from-the-file-system)
 - [Apply all templates in a directory](#apply-all-templates-in-a-directory)
 - [Apply a template from a URL](#apply-a-template-from-a-url)
-- [Apply templates from both files and URLs](#apply-templates-from-both-files-and-urls)
+- [Apply templates from files and URLs](#apply-templates-from-files-and-urls)
 - [Define environment references](#define-environment-references)
 - [Include a secret when installing a template](#include-a-secret-when-installing-a-template)
 
-{{% note %}}
-#### Apply templates to an existing stack
-To apply a template to an existing stack, include the stack ID when applying the template.
-Any time you apply a template without a stack ID, InfluxDB initializes a new stack
-and all new resources.
-For more information, see [InfluxDB stacks](/influxdb/v2.3/influxdb-templates/stacks/).
-{{% /note %}}
```

Comment:
Why remove this note? This is important when applying templates and you want to update an existing stack with an additional tempalte.

Reply:
> Why remove this note? This is important when applying templates and you want to update an existing stack with an additional tempalte.

Removed the note and moved it to https://github.com/influxdata/docs-v2/pull/4302/files#diff-b9fc01b5b06eb5bff2b5705f8c6d908b75f0ddbff4dad75f52bfba9957d6282bR242

Reply:
Oh gotcha. I just read to quickly and missed over the new section. Thanks.